### DISCUSSION TITLE
我HYP被永封了。求救

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#3959] [提问] [answered]
我跟个SB一样去作死上公益加速，新号直接秒ban，求救

**Comment by SqrtTan** [comment]
新号啊！买了才一周！

**Comment by MoYuan-CN** [comment]
去申诉，看你的封禁通知，会告诉你可以去哪里申诉的

**Comment by allMagicNB** [comment]
别急，能申诉的

**Comment by SqrtTan** [comment]
https://www.bilibili.com/video/BV1wy4y157sc/?vd_source=ab93d244b31f2732506d415228c4dcdd
我照着这个来的，先等会吧（悲

**Reply by SqrtTan** [reply]
我好像没有备注是加速ip封的，下次别给我再ban了

**Comment by wuliaodexiaoluo** [comment]
不行，我也去作个死

**Reply by SqrtTan** [reply]
6

**Reply by wuliaodexiaoluo** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/3121c352-87b8-40a8-8d62-dbcb0444ae10)
作死成功.jpg

**Comment by WForst-Breeze** [comment]
正常，去 hyp 官网申诉完等几天解封就可以了

**Reply by SqrtTan** [reply]
下次上公益加速IP还会似吗

**Reply by SqrtTan** [reply]
我买不了加速IP，只能用公益的

**Reply by WForst-Breeze** [reply]
不会了，封一次之后一般讲就不会再因为加速 IP 封禁了

**Comment by SqrtTan** [comment, marked as answer]
已成功进入14天缓冲
![image](https://github.com/Hex-Dragon/PCL2/assets/121204926/37737195-d3ca-4b9c-8262-84cfe4ebb478)
